--- raygui_buggy/src/raygui.h	2025-08-25 20:37:15.268489729 +0100
+++ raygui/src/raygui.h	2025-08-25 20:38:02.052486927 +0100
@@ -696,8 +696,8 @@
 RAYGUIAPI Font GuiGetFont(void);                                // Get gui custom font (global state)
 
 // Style set/get functions
-RAYGUIAPI void GuiSetStyle(int control, int property, int value); // Set one style property
-RAYGUIAPI int GuiGetStyle(int control, int property);           // Get one style property
+RAYGUIAPI void GuiSetStyle(int control, int property, unsigned int value); // Set one style property
+RAYGUIAPI unsigned int GuiGetStyle(int control, int property);           // Get one style property
 
 // Styles loading functions
 RAYGUIAPI void GuiLoadStyle(const char *fileName);              // Load style file over global style variable (.rgs)
@@ -1557,7 +1557,7 @@
 }
 
 // Set control style property value
-void GuiSetStyle(int control, int property, int value)
+void GuiSetStyle(int control, int property, int unsigned value)
 {
     if (!guiStyleLoaded) GuiLoadStyleDefault();
     guiStyle[control*(RAYGUI_MAX_PROPS_BASE + RAYGUI_MAX_PROPS_EXTENDED) + property] = value;
@@ -1570,7 +1570,7 @@
 }
 
 // Get control style property value
-int GuiGetStyle(int control, int property)
+unsigned int GuiGetStyle(int control, int property)
 {
     if (!guiStyleLoaded) GuiLoadStyleDefault();
     return guiStyle[control*(RAYGUI_MAX_PROPS_BASE + RAYGUI_MAX_PROPS_EXTENDED) + property];
--- raylib_buggy/src/raylib.h
+++ raylib-c/src/raylib.h
@@ -1436,7 +1436,7 @@ RLAPI void DrawTextureNPatch(Texture2D texture, NPatchInfo nPatchInfo, Rectangle
 // Color/pixel related functions
 RLAPI bool ColorIsEqual(Color col1, Color col2);                            // Check if two colors are equal
 RLAPI Color Fade(Color color, float alpha);                                 // Get color with alpha applied, alpha goes from 0.0f to 1.0f
-RLAPI int ColorToInt(Color color);                                          // Get hexadecimal value for a Color (0xRRGGBBAA)
+RLAPI unsigned int ColorToInt(Color color);                                          // Get hexadecimal value for a Color (0xRRGGBBAA)
 RLAPI Vector4 ColorNormalize(Color color);                                  // Get Color normalized as float [0..1]
 RLAPI Color ColorFromNormalized(Vector4 normalized);                        // Get Color from normalized values [0..1]
 RLAPI Vector3 ColorToHSV(Color color);                                      // Get HSV values for a Color, hue [0..360], saturation/value [0..1]
--- raylib_buggy/src/rtextures.c
+++ raylib-c/src/rtextures.c
@@ -4832,7 +4832,7 @@ Color Fade(Color color, float alpha)
 }
 
 // Get hexadecimal value for a Color
-int ColorToInt(Color color)
+unsigned int ColorToInt(Color color)
 {
     int result = 0;
 
