--- raygui_buggy/src/raygui.h	2025-08-25 20:37:15.268489729 +0100
+++ raygui/src/raygui.h	2025-08-25 20:38:02.052486927 +0100
@@ -696,8 +696,8 @@
 RAYGUIAPI Font GuiGetFont(void);                                // Get gui custom font (global state)
 
 // Style set/get functions
-RAYGUIAPI void GuiSetStyle(int control, int property, int value); // Set one style property
-RAYGUIAPI int GuiGetStyle(int control, int property);           // Get one style property
+RAYGUIAPI void GuiSetStyle(int control, int property, unsigned int value); // Set one style property
+RAYGUIAPI unsigned int GuiGetStyle(int control, int property);           // Get one style property
 
 // Styles loading functions
 RAYGUIAPI void GuiLoadStyle(const char *fileName);              // Load style file over global style variable (.rgs)
@@ -1557,7 +1557,7 @@
 }
 
 // Set control style property value
-void GuiSetStyle(int control, int property, int value)
+void GuiSetStyle(int control, int property, int unsigned value)
 {
     if (!guiStyleLoaded) GuiLoadStyleDefault();
     guiStyle[control*(RAYGUI_MAX_PROPS_BASE + RAYGUI_MAX_PROPS_EXTENDED) + property] = value;
@@ -1570,7 +1570,7 @@
 }
 
 // Get control style property value
-int GuiGetStyle(int control, int property)
+unsigned int GuiGetStyle(int control, int property)
 {
     if (!guiStyleLoaded) GuiLoadStyleDefault();
     return guiStyle[control*(RAYGUI_MAX_PROPS_BASE + RAYGUI_MAX_PROPS_EXTENDED) + property];
